README.md

```diff
@@ -43,6 +43,57 @@ property to a URL in the `static` folder of your project.
 (E.g., for `static/sfx/sound.mp3` enter `sfx/sound.mp3`).
 If `spatial` is set to `none`, all settings below are ignored.
 
+### AudioManager
+
+```js
+// Load your audio on start(), so it is ready when you need it.
+start() {
+    this.audio = globalAudioManager.load('path_to_audiofile')
+        .then((playableNode) => {
+            return playableNode;
+        });
```

Comment:
This doesn't do anything, the result is still just a promise for the playable node.